app/controllers/dead_links_controller.rb

```diff
@@ -7,6 +7,11 @@ def index
     @dead_links = DeadLink.all.includes(:funding)
   end
 
+  def update_business_types
```

Comment:
Do you really think it belongs here? I think it was at right place in the businesses_controller.

Reply:
currently the dead_links controller/view are actually cleanup. 
I can move the update_business_types method back to the businesses controller. But then there should be a cleanup-view which directs to business-controller and dead-links-controller. However, I am not sure how to have a view without an equally names controller and if that is possible at all.
The other option is to rename the dead_links view and controller alltogether to cleanup. Then the method update_business_types can remain in the cleanup-controller.

Reply:
Well, from a REST perspective, the `DeadLinksController` is named correctly because it creates and destroys dead links only. Although I get your argument about the "cleanup" naming I am not sure we should mix all that stuff in a single controller (that's not REST-like).

Maybe move the method back to the `BusinessesController` and then create a separate `CleanupController` which only contains the view you suggested (as an `index` action probably).